swastika-mtv.md

```diff
@@ -0,0 +1,5 @@
+## Mountain View, CA
+
+### Places to eat
```

Comment:
@githubteacher Maggiano's Little Italy in San Jose is great! Don't know about a great Italian restaurant in Mountain View :(